pkg/signatures/public_key_verifier.go

```diff
@@ -71,6 +71,8 @@ func newCosignPublicKeyVerifier(config *storage.CosignPublicKeyVerification) (*c
 // as well as the claim verification of the payload of the signature.
 func (c *cosignPublicKeyVerifier) VerifySignature(ctx context.Context,
 	image *storage.Image) (storage.ImageSignatureVerificationResult_Status, error) {
+	log.Debugf("Verifying signatures of image %q with the following public keys: %+v",
```

Comment:
One problem I have with introducing such logs is the potential pollution of our development logs, which would render them pretty useless(or we would need to start using some tool to efficiently look at logs). In my personal experience, our logs seem pretty polluted at this point, even at a non-debug level.

I like the logs introduced here as a preventive measure. They would help us to investigate in case there are some issues with the image signing feature shortly after release. However, I think long-term we should think not about adding more logs, but rather about removing useless log statements.

My solution to this would be to add these logs and create a ticket which we should be able to pick up in a month or two and see if there is a good reason to keep/remove these logs. My concern with this solution is the feeling that we are overusing the "see a problem - put it in a backlog" approach. However, I don't see what else can we do at this point to determine whether these logs will worsen the developer experience or not. WDYT?

Reply:
Hm, I do agree with the sentiment of overusing the "put problem in backlog approach", but as you mentioned, I also don't really see another solution w.r.t the developer experience 😞 
I've added [ROX-9973](https://issues.redhat.com/browse/ROX-9973) which we can revisit after the release to check back on it.